README.md

```diff
@@ -1,25 +1,74 @@
+- [ ] (1) Tests are included and:
+  - Are readable (intent is clear)
+  - Cover happy path
+  - Cover edge cases
+
+### Documentation
+
+- [ ] (2) Clear instructions for project setup
+- [ ] (-1) Clear code comments, but _only when necessary_. Comments are for the why, not the what
 
 ## Git Hygiene
-- Commits tend towards atomic (take care of a single thing)
-- Commit messages are descriptive and follow the [seven rules of a great Git commit message](https://chris.beams.io/posts/git-commit/)
-- Pull requests are atomic (take care of a single thing) and functional (don’t leave the program in a half-functioning state)
-- Pull requests have proper description, mentioning the what and the why and (infrequently and if complex) the how, and include detailed test plans (steps + expected outcome)
-
-## Functionality
-- All task requirements are fully implemented, tradeoffs made are documented, explanations are provided for any missing pieces
-- Not obviously broken
-- No dubious functionality on top of the spec
-- Not over-engineered, e.g., doesn’t prematurely optimize at the expense of complexity
-
-## Quality
-- Code is self-documenting: semantic variables names, function names describe implemented function
-- Code is as simple as possible and modular
-- Small single-purpose functions > giant functions that do 10 things
-- Code is logically separated (e.g., no giant utils folders)
-- Author uses objects, arrays, inheritance, closures, callbacks, promises expertly, and is up to date with the language/technology, e.g., async/- await > promise chains > callbacks for code after 2019
-- Tests are included and:
-    - Are readable (intent is clear)
-    - Cover happy path
-    - Cover edge cases
\ No newline at end of file
+
+> [!IMPORTANT]
+>
+> - Candidates should score at least 5 points in this section (or skip if not working with Pull Requests, e.g. in the context of a code challenge)
+> - Colleagues should score at least 10 points in this section
+
+- [ ] (1) Commits tend towards atomic (take care of a single thing)
+- [ ] (1) Commit messages are descriptive and follow the [seven rules of a great Git commit message](https://chris.beams.io/posts/git-commit/)
+
+If using Pull Requests:
+
+- [ ] (1) Pull requests are atomic (take care of a single thing) and functional (don’t leave the program in a half-functioning state)
+- [ ] (1) Pull requests have proper description, mentioning the what and the why and (infrequently and if complex) the how, and include detailed test plans (steps + expected outcome)
+- [ ] (1) Knows how to rebase commits
+- [ ] (1) Knows how and when to squash/fixup commits
+- [ ] (1) Pull requests follow the Pull Request template in the repo (if one is present)
+- [ ] (1) Knows how to create and maintain (through rebasing) a chain of related pull requests for easier review and merging (Bonus for candidates, requirement for colleagues)
+
+## Code Reviews
```

Comment:
it would be cool!

Reply:
Actually we could do that.  We are talking about our expedited challenge but what if we went the other way and said "you have 5 days to implement X.  Please make a PR for these features" -- might be too complicated though especially because we'd have to be on the ball with reviews